builders/ethereum/verify-contracts/etherscan-plugins.md

```diff
@@ -20,12 +20,14 @@ To follow along with this tutorial, you will need to have:
 - [MetaMask installed and connected to the Moonbase Alpha](/tokens/connect/metamask/){target=\_blank} TestNet
 - An account funded with `DEV` tokens.
  --8<-- 'text/_common/faucet/faucet-list-item.md'
-- A Moonscan API key for the network you're trying to verify a contract on. For Moonbeam and Moonbase Alpha, you'll need a [Moonbeam Moonscan](https://moonscan.io){target=\_blank} API key. For Moonriver, you'll need a [Moonriver Moonscan](https://moonriver.moonscan.io){target=\_blank} API key
+- An Etherscan API key
 - Git installed and configured
 
-## Generating a Moonscan API Key {: generating-a-moonscan-api-key }
+## Generating an Etherscan API Key {: generating-an-etherscan-api-key }
 
-To generate a Moonscan API Key, you will need to sign up for an account. Depending on which network you want to verify a contract on, you'll have to make sure you create an API key from the correct network on Moonscan. For Moonbeam or Moonbase Alpha, you can navigate to the [Moonbeam Moonscan](https://moonscan.io){target=\_blank}. For Moonriver, you can head to the [Moonriver Moonscan](https://moonriver.moonscan.io){target=\_blank}. To sign up for an account, you can take the following steps:
+To verify contracts on Moonscan for any Moonbeam network, you’ll need [an Etherscan API key](https://docs.etherscan.io/etherscan-v2/getting-an-api-key){target=\_blank}. Because Moonscan belongs to Etherscan’s ecosystem of 50-plus chains, a single key works across all supported networks.
```

Comment:
```suggestion
To verify contracts on Moonscan for any Moonbeam network, you’ll need [an Etherscan API key](https://docs.etherscan.io/etherscan-v2/getting-an-api-key){target=\_blank}. Because Moonscan belongs to Etherscan’s ecosystem, a single key works across all supported networks.
```

What if this amount changes? I think it's fine to just drop the number of chains here.